Matches

nepito opened the pull request.

Commit:
🧪🛑 Test the class Matches

Commit:
⚗️☂️ Try send coverage results

Commit:
👩🏿‍💻 Add file to play with the class Matches

Commit:
✅🗃️ Add data from the bundesleague

Commit:
🤐✅Mute some failing tests

Commit:
🎨Make format with styler

Commit:
🔨 Make setup_github change to run in guthub

Commit:
✅📝🗃️Add data for Serie A 2020

Commit:
✅📝🗃️Add data for Ligue 1 2020

Commit:
✅📝🗃️Add data of the week

davidmacer commented on src/calculate_xGoal.R:
```diff
@@ -1,7 +1,7 @@
 source("R/xTable.R")
 
-league <- read_csv("tests/data/statistics_39_2020.csv")
+league <- read_csv("tests/data/statistics_61_2020.csv")
```
¿Por qué cambiaste el `statistics_39_2020.csv` por `statistics_61_2020.csv`? ¿En algún momento pondrás de nuevo la tabla de datos que borraste?

Commit:
✅📝🗃️Add data of the MX

Commit:
✅📝🗃️ Add penalties data for Serie A 🇮🇹

Commit:
✅📝🗃️ Add penalties data for MX 2021 🇲🇽

Commit:
✅📝🗃️ Add penalties data for MX 2020 🇲🇽

Commit:
👩🏿‍💻 Update xGoal using penalties data

Commit:
✅📝🗃️ Add penalties data for Serie A 🇮🇹

Commit:
✅📝🗃️ Add penalties data for Bundesliga 2021 🇩🇪

Commit:
👩🏿‍💻 Add xGoal for penalties

Commit:
👩🏿‍💻 Add xGoal for penalties

Commit:
✅📝🗃️Add data of the Eredivisie 🇳🇱

Commit:
✅📝🗃️Add data of the Eredivisie 2021 🇳🇱

Commit:
✅📝🗃️Add data of La Liga 2021 🇪🇸

Commit:
✅📝🗃️Add data of penalties of La Liga 2020 🇪🇸

Commit:
✅📝🗃️Add data of penalties of Premier 2021 🏴󠁧󠁢󠁥󠁮󠁧󠁿

Commit:
✅📝🗃️Add data of penalties of Premier 2020 🏴󠁧󠁢󠁥󠁮󠁧󠁿

Commit:
✅📝🗃️Add data of the Primeira Liga 2020 🇵🇹

Commit:
👩🏿‍💻 Add new script to CTF

Commit:
🤺👩🏿‍💻 Add fitters

Commit:
🔥 Delete fit of the model

Commit:
✅📝🗃️Add data and names of teams

Commit:
👩🏿‍💻 Add variables about penalties

Commit:
✅📝🗃️Update data of 🇩🇪🇪🇸🏴󠁧󠁢󠁥󠁮󠁧󠁿

Commit:
✅📝🗃️Update data of  🏴󠁧󠁢󠁥󠁮󠁧󠁿

Commit:
🧪🛑Test function previous_season

Commit:
😇 Pass test

Commit:
🧪🛑 Add second test

Commit:
✅ Pass the test

Commit:
✅ Assess the function previous_season

Commit:
⚡ Add files to predictions

Commit:
👩🏿‍💻 Add cli

Commit:
👩🏿‍💻 Add cli

Commit:
👩🏿‍💻 Show results

Commit:
👩🏿‍💻➕ Add correct dependency

Commit:
👩🏿‍💻 Read the variable from cli

Commit:
📝🗃️ Add data from leagues 🇩🇪🇪🇸🏴󠁧󠁢󠁥󠁮󠁧󠁿🇮🇹

Commit:
📝 Update readme

Commit:
♻️ Refactor functions

Commit:
📝🗃️ Add data from leagues 🇳🇱🇵🇹🇩🇪🇪🇸🏴󠁧󠁢󠁥󠁮󠁧󠁿🇮🇹🇫🇷

Commit:
📝🗃️ Add data from leagues 🇳🇱🇵🇹🇩🇪🇪🇸🏴󠁧󠁢󠁥󠁮󠁧󠁿🇮🇹🇫🇷

Commit:
📝🗃️ Add data from leagues 🇳🇱🇵🇹🇪🇸🏴󠁧󠁢󠁥󠁮󠁧󠁿🇮🇹🇫🇷

Commit:
🔨 Add recipe phony to update strength

Commit:
📝🗃️ Add data from leagues 🇳🇱🇵🇹🇩🇪🇪🇸🏴󠁧󠁢󠁥󠁮󠁧󠁿🇮🇹🇫🇷

Commit:
👩🏿‍💻 Add fast api

Commit:
📝🗃️ Add data from leagues 🇳🇱🇵🇹🇩🇪🇪🇸🏴󠁧󠁢󠁥󠁮󠁧󠁿🇮🇹🇫🇷

Commit:
👩🏿‍💻 Parametrize probabilities

Commit:
👩🏿‍💻 Choose last round

Commit:
📝🗃️ Add data from league 🇩🇪

Commit:
✏️🔨 Fix typo

Commit:
📝🗃️ Add data from league 🏴󠁧󠁢󠁥󠁮󠁧󠁿

Commit:
📝🗃️ Add data from leagues 🇳🇱🇵🇹🇩🇪🇪🇸🏴󠁧󠁢󠁥󠁮󠁧󠁿🇮🇹🇫🇷

Commit:
📝🗃️ Add data from leagues 🇳🇱🇵🇹🇩🇪🇪🇸🏴󠁧󠁢󠁥󠁮󠁧󠁿🇮🇹🇫🇷

Commit:
📝🗃️ Add data from leagues 🇳🇱🇵🇹🇩🇪🇪🇸🏴󠁧󠁢󠁥󠁮󠁧󠁿🇫🇷

Commit:
📝🗃️ Add data from league 🇮🇹

Commit:
🐛📝🗃️ Dix bug, add penalties info to data from league 🇮🇹

Commit:
🙅🏿‍♀️ 🚟 Do not  use streak

Commit:
🚧 Add results the the api

Commit:
📝🗃️ Add data from leagues 🇳🇱🇵🇹🇩🇪🇪🇸🏴󠁧󠁢󠁥󠁮󠁧󠁿🇮🇹🇫🇷

Commit:
🎨 Format

Commit:
🐋➕ Add deta

Commit:
📝🗃️ Add data from leagues 🇳🇱🇵🇹🇩🇪🇪🇸🏴󠁧󠁢󠁥󠁮󠁧󠁿🇮🇹🇫🇷🇲🇽

Commit:
🔨 Update recipe `update_premier_league`

Commit:
✅Update tests "The function calculate_xgoal"

La interfaz de la función `calculate_xgoal()` cambió. Ahora necesita los tiros de panles

Commit:
✅ Pass tests

Commit:
⚗️🐋 Change instructions to download packages

Commit:
🐋Update parent image

Commit:
⚡️ Add directory option

Commit:
⚡️ Use join functions

Commit:
🔨 Use new interface

Commit:
🔧 Update DESCRIPTION

Commit:
⚗️🪲🐋 Fix problem in the installation

Commit:
⚡Add silent version

Commit:
⚡Add silent version

Commit:
✅ Fix test
